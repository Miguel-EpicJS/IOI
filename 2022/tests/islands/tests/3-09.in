4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
100 198
0 31
31 0
37 86
86 37
49 81
81 49
18 65
65 18
13 30
30 13
88 71
71 88
67 21
21 67
55 37
37 55
76 62
62 76
99 69
69 99
66 90
90 66
5 38
38 5
68 59
59 68
33 96
96 33
81 15
15 81
32 22
22 32
50 88
88 50
14 52
52 14
59 17
17 59
42 84
84 42
26 11
11 26
75 99
99 75
54 57
57 54
84 36
36 84
11 67
67 11
51 25
25 51
47 58
58 47
9 97
97 9
2 55
55 2
36 45
45 36
39 49
49 39
61 85
85 61
93 64
64 93
65 14
14 65
71 16
16 71
90 32
32 90
16 47
47 16
72 66
66 72
91 35
35 91
63 7
7 63
83 91
91 83
31 26
26 31
61 75
75 61
74 60
60 74
7 6
6 7
28 20
20 28
58 46
46 58
29 33
33 29
46 12
12 46
10 51
51 10
30 53
53 30
25 61
61 25
96 28
28 96
38 44
44 38
1 29
29 1
60 40
40 60
57 1
1 57
34 39
39 34
40 9
9 40
89 34
34 89
94 78
78 94
98 42
42 98
19 23
23 19
73 18
18 73
48 24
24 48
62 5
5 62
15 76
76 15
77 92
92 77
82 74
74 82
4 10
10 4
69 48
48 69
52 3
3 52
97 41
41 97
56 80
80 56
12 27
27 12
86 68
68 86
78 89
89 78
95 98
98 95
21 4
4 21
41 56
56 41
8 77
77 8
23 73
73 23
3 82
82 3
27 87
87 27
35 54
54 35
45 79
79 45
92 83
83 92
79 43
43 79
53 70
70 53
44 13
13 44
6 93
93 6
70 95
95 70
22 8
8 22
17 63
63 17
43 2
2 43
20 50
50 20
87 19
19 87
24 94
94 24
64 72
72 64
