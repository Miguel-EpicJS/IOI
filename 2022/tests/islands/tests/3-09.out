91041670623a69783795ed82bda33426101ce311
OK
1
36
0 82 40 48 12 156 138 98 50 102 62 63 84 42 18 19 42 84 63 62 84 42 19 18 42 84 102 50 98 138 156 12 48 40 82 0
